### DISCUSSION TITLE
Overclock Settings Question

### DISCUSSION THREAD

**Discussion by garland410** [rg35xx-cfw/rg35xx-cfw.github.io#88] [Q&A]
Might be a dumb question but once enabled, how do I turn it off? Is medium setting off? Nebbing around the web and settings from the original rg35xx make me think that might be the case but would like to be sure.

**Comment by acmeplus** [comment]
In emulationstation, go to Settings, System Settings, and then Overclock. By default it's off but the governor is in schedutil mode that basically will go up and down the frequencies and needed by the processes running. However, you can use that menu to set it at a specific frequency. Note that the Extreme (1512 MHz) is perfectly safe in this console, so you can use that for demanding games where battery is not the higher priority.

**Reply by davidpasion** [reply]
I think what @garland410 was asking was is there a way to revert back to the "NONE" overclock option where no specific frequency is selected and it changes according to demand like you mentioned. Once we select low, medium, or extreme, it stays that way until we change it (at least on the 37-dev version from May 2023). I just checked your January 2024 release and there aren't even any options besides "NONE" on the overclock menu. Does that mean the January 2024 release's processor frequencies are always on auto now?

**Comment by garland410** [comment]
Is there a way to put it back in the default mode though? Once changed?
